airbyte-workers/src/test/java/io/airbyte/workers/tracing/DummySpan.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.tracing;
+
+import datadog.trace.api.interceptor.MutableSpan;
+import java.util.HashMap;
+import java.util.Map;
+
+class DummySpan implements MutableSpan {
+
+  @SuppressWarnings("PMD")
```

Comment:
@colesnodgrass Any insight as to why this suppression was added?  @benmoriceau This was a copy/extraction from the existing test, so I don't have the context.

Reply:
I meant to clean this up before committing it. Since there was a 30 minute turn around time between making a change and getting it into dev to verify it, I didn't want the build failing due to a PMD issue (making that 30 minutes even longer). Once things were working as expected I should have fixed this to suppress only the specific issue (if any even exist anymore).

Reply:
I did some testing locally and this suppression does not appear to be necessary.  I've removed it and will verify that the PR builds without issue.

Reply:
Looks like it could be removed safely.